packages/strapi/lib/services/metrics/sender.js

```diff
@@ -57,10 +57,8 @@ module.exports = strapi => {
         event,
         uuid,
         deviceId,
-        properties: {
-          ...payload,
-          ...anonymous_metadata,
-        },
+        // every property must be a string
+        properties: _.mapValues({ ...payload, ...anonymous_metadata }, _.toString),
```

Comment:
Can payload and anonymous_metadata contain objects ?

![Capture d’écran 2021-04-21 à 15 40 34](https://user-images.githubusercontent.com/11708220/115563632-0d8da180-a2b8-11eb-9822-8f7c8833b24b.png)

Reply:
arrays actually nice catch